INSTALL.md

```diff
@@ -194,7 +188,121 @@ Run [NPM](https://www.npmjs.com/) to install all the required Node modules:
 npm install
 ```
 
-#### Setup Database
+## Manual setup on Fedora (29)
+
+### Prerequisites
+
+#### Ruby version
+
+The project supports **Ruby 2.3.x**.
+
+Verify you have a proper version by running on your terminal:
+```bash
+ruby -v
+```
+
+> You can use [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/) to install ruby.
+
+#### Node version
+
+The project supports **Version: 8.X.X**.
+
+You might want to use [nvm](https://github.com/creationix/nvm/) to install and work with specific Node versions:
```

Comment:
OK so this is not enforcing the use of NVM, probably say then to install version 8 of nodejs and ensure that environment variables like `PATH` are setup correctly